tests/feature_elimination/test_feature_elimination.py

```diff
@@ -73,7 +82,9 @@ def test_shap_rfe(X, y, capsys):
             scoring="roc_auc",
             n_jobs=4,
         )
-        shap_elimination = shap_elimination.fit(X, y, approximate=True, check_additivity=False)
+        shap_elimination = shap_elimination.fit(
+            X, y, sample_weight=sample_weight, approximate=True, check_additivity=False
```

Comment:
I think you still missed approximate=False here.

Reply:
That one is a `DecisionTreeClassifier` and doesn't need `approximate=False`. The one where it fails is in `test_model_interpret` - I missed it, but find it curious that it ran before - for me LGBM with categorical features would fail before. I am pushing a fix for it anyways.